### ISSUE TITLE
Consistent versioning 

### ISSUE THREAD

**Issue by myhau** [Azure/secrets-store-csi-driver-provider-azure#165]
**Describe the solution you'd like**

Helm chart with version 0.7.x should only deploy csi provider with versions 0.7.x (not 0.6.x, not 0.8.x).

**Why**

We lost a lot of time debugging an issue fixed in 0.7.0 thinking that Helm chart with version 0.7.0 deploys csi with version 0.7.0. 

**Alternative**

Helm chart version / csi version table in `README.md`

**Comment by aramase** [comment]
@myhau The `chartVersion` and `appVersion` aren't required to be in sync. When there is chart only updates like updating the driver dep, adding new fields to the chart, then only the `chartVersion` is bumped. When there is a new release of the Key Vault provider, then the `appVersion` is bumped. 

Running `helm ls` provides the current app version -

```
➜ helm ls
NAME	NAMESPACE	REVISION	UPDATED                             	STATUS  	CHART                                 	APP VERSION
csi 	default  	1       	2020-07-20 09:51:38.056077 -0700 PDT	deployed	csi-secrets-store-provider-azure-0.0.8	0.0.7
```

**Comment by myhau** [comment]
Sorry, I was a bit frustrated when writing this. You are right, `chartVersion` and `appVersion` aren't required to be in sync. 

<img width="343" alt="Screenshot 2020-07-22 at 10 48 34" src="https://user-images.githubusercontent.com/4415632/88155842-e4e15a00-cc08-11ea-9abf-f52897375a85.png">

^ But, seeing this, for some reason made me expect that by installing Helm chart with version `0.0.7` I will end up with `0.0.6` csi driver. I noticed it (by checking pod's image), because `0.0.6` has some bug that I knew was fixed in `0.0.7`. 

Having simple version matrix in your `README.md` would be great:

|Helm chart version|Driver version|
|-|-|
|`0.0.8`|`0.0.7`|
|`0.0.7`|`0.0.6`|
|....|....

**Comment by ritazh** [comment]
@myhau Thanks for the feedback! We will add it to the readme.
